.github/workflows/main.yml

```diff
@@ -20,4 +20,6 @@ jobs:
           port: ${{ secrets.PORT }} 
           script: | 
             cd sugarizer-dev/sugarizer 
-            git pull
\ No newline at end of file
+
```

Comment:
This file should not be a part of the PR.

Reply:
I have deleted this File